### PR TITLE
added is_pithreeplus() - closes #374

### LINKED ISSUE #374: Detect RPi3B+
https://community.openhab.org/t/raspberry-3-b-wifi-on-openhabian/43417

@mues-lee could you troubleshoot this problem for me?

My guess: This routine doesn't match the new Pi, hence that condition isn't true:

https://github.com/openhab/openhabian/blob/master/functions/helpers.sh#L55
https://github.com/openhab/openhabian/blob/master/functions/wifi.sh#L5

### PR DESCRIPTION
**Pull request by mues-lee**
closes #374

could been done also by just adjusting the regex form is_pithree() and the differentiation in pi3b and pi3b+ is not really needed at the moment, but is "a nice to have"-feature  for future use

### CODE REVIEW

File: functions/system.sh [hunk 1]
```diff
@@ -276,7 +276,7 @@ Finally, all common serial ports can be made accessible to the openHAB java virt
 
   # Find current settings
   if is_pi && grep -q "enable_uart=1" /boot/config.txt; then sel_1="ON"; else sel_1="OFF"; fi
-  if is_pithree && grep -q "dtoverlay=pi3-miniuart-bt" /boot/config.txt; then sel_2="ON"; else sel_2="OFF"; fi
+  if ( is_pithree || is_pithreeplus ) && grep -q "dtoverlay=pi3-miniuart-bt" /boot/config.txt; then sel_2="ON"; else sel_2="OFF"; fi
```
Comment (ThomDietrich) [inline functions/system.sh]: why braces here?

**Comment by ThomDietrich** [comment]
I'm also not sure when and how this could be useful. Now that you went the extra mile I' okay to do it this way.

**Comment by mues-lee** [comment]
most likely will will never need it

also i noticed that I somehow managed to not branch the master, but my own webif branch, sorry for this
maybe it will be better to discard this PR, to keep everything separated and clean

File: functions/system.sh (hunk 1 above)
Comment (ThomDietrich) [reply functions/system.sh]: Ah I see. In bash this is in fact not needed.

https://unix.stackexchange.com/a/88851

**Comment by ThomDietrich** [comment]
Yes I saw that. A new PR would be the easiest solution, yeah

**Comment by mues-lee** [comment]
PR will be back in a clean state

File: functions/system.sh (hunk 1 above)
Comment (mues-lee) [reply functions/system.sh]: i will change this in the new PR

**Pull request closed without merging**